eclipse-scout-core/src/table/TableTileGridMediator.ts

```diff
@@ -294,7 +294,7 @@ export class TableTileGridMediator extends Widget implements TableTileGridMediat
   protected _createTileAccordion(): TileAccordion {
     return scout.create(TileAccordion, {
       parent: this.table,
-      virtual: true,
+      virtual: false,
```

Comment:
I don't think it is good to change the default behavior of all TileAccordions created by the TableTileGridMediator to "not-virtual". This should be set via a property only where it is needed (analogue gridColumnCount).